app/controllers/forms_controller.rb

```diff
@@ -16,4 +23,8 @@ def update
       render "question_flow/#{step}"
     end
   end
+
+  def show_early_result_screen?
```

Comment:
This feels like we are calling the feature flag too often. The flow model revolves around 'completing' screens, which I think we need to keep in step with. I think it might be worth adding an extra 'Form' for the interruption screen, that 'completes' when visited (if that is the required behaviour) and is shown when ineligible_gross_income is true. This form could then be inserted into the flow only when the feature flag is true. WDYT?

Reply:
I would be interested to talk this through and see what it looks like, I think adding a form could be a good idea. Agree we are calling the flag a lot and if we can find a way to do that less/more cleanly, that would be good.

Reply:
I'm slightly concerned that adding the interruption screen might be a bit 'too simple' - I need to think some more